lib/rubocop/extension/generator/generator.rb

```diff
@@ -136,8 +150,8 @@ def self.defaults!
             end
           RUBY
 
-          patch 'Gemfile', /\z/, <<~RUBY
-            gem 'rspec'
+          patch 'Gemfile', /^(\s*gem\s['"]rspec['"].*)$/, <<~RUBY
```

Comment:
Creating a new gem with `bundle gem` and RSpec as the test framework adds the rspec gem to the Gemfile by default, so patching a newline to add `gem 'rspec'` makes the `bundle install` command to fail. This change updates the Gemfile, in the line that contains `gem 'rspec'` to just replace whatever is for `gem "rspec"` (notice it removes the gem version, if any).